### ISSUE TITLE
[Suggestion] Make the Moon render independently from the Sun

### ISSUE THREAD

**Issue by Jerhom3** [Nettakrim/Spyglass-Astronomy#23]
I love the mod, but, what if the moon could also cause a, purely cosmetic, eclipse to happen in game, something along the likes of the Caelum mod on forge

**Comment by Nettakrim** [comment]
yeah ive also been considering having a second moon have a chance to exist in a world, and its not like moon phases make any sense normally anyway

(i headcannon that theres a giant mirror in orbit, since there should always be lunar eclipse)

**Comment by AlienXtream** [comment]
i think actually decoupling the moon and sun could be cool. moon phases and such would be a problem though as those affect some things (slimes in swamps) in game. it may be better to have that feature be a separate serverside companion mod.

**Comment by Nettakrim** [comment]
spyglass astronomy will never do this because its designed to sit on top of vanilla instead of overhauling anything

this mod that someone else made: https://modrinth.com/mod/astrocraft aims for more realism, so it does this, and may be what you want
